package.json

```diff
@@ -33,7 +33,7 @@
     "@capacitor/status-bar": "6.0.0",
     "@ionic/angular": "^7.0.0",
     "@zxing/browser": "^0.1.5",
-    "authentication_frontend_library": "^0.0.62",
+    "authentication_frontend_library": "^0.0.63",
```

Comment:
Why is it changed to 0.0.63?
Any changes or updates done?

Reply:
the emit from the logout is been done in the this version